massa-node/base_config/config.toml

```diff
@@ -367,17 +367,17 @@
     # [optional] port on which to listen for incoming bootstrap requests. You may need to change this to "0.0.0.0:port" if IPv6 is disabled system-wide.
     bind = "[::]:31245"
     # timeout to establish a bootstrap connection
-    connect_timeout = 15000
+    connect_timeout = 150000000000
     # timeout for providing the bootstrap to a connection
-    bootstrap_timeout = 1200000
+    bootstrap_timeout = 120000000000000
```

Comment:
I think we can't merge with those limits for connect_timeout and bootstrap_timeout, is it possible to narrow down the acceptable values and explain how those new limits are computed?

Reply:
Does the bootstrap timeout limit corresponds roughly to 6j 11h 31m 12s?

Reply:
No, I just added a bunch of zero
For now we could put a timeout of 1 day, that corresponds to `86400000` milliseconds.
I'll add the `update bootstrap timeout` on the list of issues regarding the bootstrapping process

Reply:
And I'll reduce the other timeouts as well

Reply:
Perfect, this seems good to me